src/test/java/io/unlaunch/automation/pages/AllTest.java

```diff
@@ -51,74 +52,82 @@ public void sleepBeforeEachTest() {
     }
 
     @Test
-    @Order(1)
+    @Order(10)
     public void testFillNewRegistrationForm() throws IOException{
         registration.deleteUserBeforeRegister();
         registration.register();
     }
 
     @Test
-    @Order(2)
+    @Order(20)
     public void testRegisterOtp() {
         registration.otp();
     }
 
     @Test
-    @Order(3)
+    @Order(30)
     public void testRegisterCompany() {
         registration.company();
     }
 
     @Test
-    @Order(4)
+    @Order(40)
     public void testRegisterProject() {
         registration.createFirstProject();
     }
 
     @Test
-    @Order(5)
+    @Order(50)
     public void testRegisterSkipMembers() {
         registration.skipMembers();
     }
 
     @Test
-    @Order(6)
+    @Order(60)
+    public void tutorialSuccess() {
+        tutorial.apiEventReceived();
+    }
+
+    /*
+    @Test
+    @Order(70)
     public void testVerifySuccessfulLogin() {
-        registration.verifySuccessfullLogin();
+        registration.verifySuccessfulLogin();
     }
+     */
 
     @Test
-    @Order(7)
+    @Order(80)
```

Comment:
why is order changed?

Reply:
so we can add more tests in without changing everything all the order numbers.